historical_data/src/get_previous_day_data.py

```diff
@@ -46,26 +49,41 @@ def get_historical_data(stocks: pd.Series, day) -> pd.DataFrame:
         inplace=True
     )
     df['date'] = day
-    print(df.columns)
-    print("data frame is:\n{}".format(df))
+    # print(df.columns)
+    # print("data frame is:\n{}".format(df))
     return df
 
 
-def get_prev_bday_data(stocks: pd.Series) -> pd.DataFrame:
+def get_prev_bday_data() -> pd.DataFrame:
     """Retrieve info for previous business day"""
     today = date.today()
     prev_bday = today - BDay(1)
-    return get_historical_data(stocks, prev_bday)
+    print(prev_bday)
+    query = '''
+    select stocks.ticker from stocks left join (select historical_data.ticker, max(date) as recent_date
+    from historical_data GROUP BY ticker HAVING recent_date='{pull_date}') B on stocks.ticker=B.ticker where B.ticker IS null; 
+    '''.format(pull_date=prev_bday)
+    tickers_from_db = pd.read_sql_query(
+        query, conn
+    )
+    stock_list = tickers_from_db['ticker'].tolist()
+    if stock_list:
+        stocks = pd.Series(tickers_from_db['ticker'].tolist())
```

Comment:
This returns pandas Series directly.
```suggestion
        stocks = tickers_from_db['ticker']
```